_people/progyan.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Progyan Bhattacharya
+img: pg.png
+nickname: GOD
```

Comment:
Unfortunately there can only be one God, and He is requesting you to choose a different nickname

Reply:
😆